dbt_project.yml

```diff
@@ -1,5 +1,5 @@
 name: 'trino_utils'
 version: '0.3.0'
 config-version: 2
-require-dbt-version: [">=1.3.0", "<2.0.0"]
+require-dbt-version: [">=1.4.0", "<2.0.0"]
```

Comment:
Why the need to upgrade?

This would mean that dbt 1.3 users can't use this package.

Why not setup a test matrix?

Reply:
Right, I reverted this change. I will create separate issue for creating test matrix for different dbt versions.